services/payments/src/simcore_service_payments/services/rabbitmq.py

```diff
@@ -56,6 +57,10 @@ def get_rabbitmq_client(app: FastAPI) -> RabbitMQClient:
     return cast(RabbitMQClient, app.state.rabbitmq_client)
 
 
+def get_rabbitmq_client_from_request(request: Request):
```

Comment:
same here but it seems @pcrespov started to change the pattern...
services/payments/src/simcore_service_payments/api/rest/_dependencies.py